components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/Utils.java

```diff
@@ -54,12 +55,23 @@ private Utils() {
     /**
      * This function validates the JWT token by its content using nimbus libraries.
      *
-     * @param idToken  String. The jwt token string.
-     * @param audience String. Authenticator client ID to check validity.
+     * @param idToken            String. The jwt token string.
+     * @param audience           String. Authenticator client ID to check validity.
```

Comment:
Formatting issues. Please check the other places as well.

Reply:
Fixed dbe9564e248b273f971fd3604532278e80046c07